test/NuGet.Core.Tests/NuGet.Commands.Test/SignCommandRunnerTests.cs

```diff
@@ -108,7 +108,15 @@ public async Task ExecuteCommandAsync_WithAmbiguousMatch_ThrowsAsync()
             using (var test = await Test.CreateAsync(_fixture.GetDefaultCertificate()))
             {
                 test.Args.CertificateSubjectName = "Root";
-                test.Args.CertificateStoreLocation = StoreLocation.LocalMachine;
+                if (RuntimeEnvironmentHelper.IsWindows || RuntimeEnvironmentHelper.IsMacOSX)
+                {
+                    test.Args.CertificateStoreLocation = StoreLocation.LocalMachine;
+                }
+                else if (RuntimeEnvironmentHelper.IsLinux)
+                {
+                    test.Args.CertificateStoreLocation = StoreLocation.CurrentUser;
+                }
+                
```

Comment:
uhhh, didn't one of the previous things make the store CurrentUser for MacOSX? revisit this.

Reply:
Thanks! Pls refer to https://github.com/NuGet/NuGet.Client/pull/3168#discussion_r362672651